engine/access/rest/converter.go

```diff
@@ -32,30 +59,99 @@ func toID(id string) (flow.Identifier, error) {
 	return flowID, nil
 }
 
+func toIDs(ids string) ([]flow.Identifier, error) {
+	// currently, the swagger generated Go REST client is incorrectly doing a `fmt.Sprintf("%v", id)` for the id slice
+	// resulting in the client sending the ids in the format [id1 id2 id3...]. This is a temporary workaround to
+	// accommodate the client for now. Issue to to fix the client: https://github.com/onflow/flow/issues/698
+	ids = strings.TrimSuffix(ids, "]")
+	ids = strings.TrimPrefix(ids, "[")
+
+	reqIDs := strings.Split(ids, ",")
```

Comment:
I'm starting to think supporting any other format is a mistake, the client is responsible to correctly format the request params. This was only done because auto-generated go client is creating this kind of request but that doesn't mean anything, it just means the generated client (as any other generated code from that lib) is not good.

Reply:
I think it would be totally fine to only support the correct encoding, and document somewhere that the auto generated client code has this issue